### PR TITLE
Docs: Plugin Installation - reorganize and add air-gapped environments

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What is this feature?**
Documentation improvements:
- Reduce redundant plugin installation information
- Add guidance for air-gapped environments

**Why do we need this feature?**

Improve documentaiton

**Who is this feature for?**
Grafana Admins

**Which issue(s) does this PR fix?**:

Partially address https://github.com/grafana/grafana/issues/79625

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: docs/sources/administration/plugin-management/index.md [hunk 1]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
 
 ### Install a plugin
 
-To install a plugin:
+To install a plugin you have multiple possibilities, the regular way is using Grafana UI.
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:106]: The most common way to install a plugin is through the Grafana UI, but alternative methods are also available.

File: docs/sources/administration/plugin-management/index.md [hunk 2]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
...
-### Update a plugin
+There are also additional ways to install plugins depending on your setup.
 
-To update a plugin:
-
-1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
-1. Click the **Installed** filter to show only installed plugins.
-1. Click the plugin's logo.
-1. Click **Update**.
+#### Install a plugin using grafana cli
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:115]: Install a plugin using Grafana CLI

File: docs/sources/administration/plugin-management/index.md [hunk 3]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
...
+Grafana CLI allows you to install, upgrade, and manage your Grafana plugins using a command line. For more information about Grafana CLI plugin commands, refer to [Plugin commands]({{< relref "../../cli/#plugins-commands" >}}).
 
-When the update is complete, you'll see a confirmation message that the update was successful.
 
-### Uninstall a plugin
+#### Install a plugin from a zip file
+This way is usually used for plugins not distributed via plugins catalog or in environments without internet connection.
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:120]: This method is typically used for plugins not available in the Plugin Catalog or in environments without internet access.

File: docs/sources/administration/plugin-management/index.md [hunk 4]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
...
-1. In a web browser, navigate to the [Grafana plugin catalog](https://grafana.com/plugins) and find a plugin that you want to install.
-1. Click the plugin, and then click the **Installation** tab.
+The path to the plugin directory is defined in the configuration file. For more information, refer to [Configuration]({{< relref "../../setup-grafana/configure-grafana/#plugins" >}}).
 
-### Install plugin on Grafana Cloud
+#### Install a plugin in air-gapped environment
+Plugin installation usually requires an internet connection, you can check which endpoints are being used on your instance when installing a plugin regularly and add them to allowlist of your instance.
 
-On the **Installation tab**, in the **For** field, click the name of the Grafana instance on which you want to install the plugin.
+If this is not possible you can go via installing a plugin using [grafana cli](#install-a-plugin-using-grafana-cli) or as a [zip file](#install-a-plugin-from-a-zip-file).
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:133]: If this is not possible you can go via installing a plugin using the [Grafana CLI](#install-a-plugin-using-grafana-cli) or as a [ZIP file](#install-a-plugin-from-a-zip-file).

File: docs/sources/administration/plugin-management/index.md [hunk 5]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
...
+Grafana CLI allows you to install, upgrade, and manage your Grafana plugins using a command line. For more information about Grafana CLI plugin commands, refer to [Plugin commands]({{< relref "../../cli/#plugins-commands" >}}).
 
-When the update is complete, you'll see a confirmation message that the update was successful.
 
-### Uninstall a plugin
+#### Install a plugin from a zip file
+This way is usually used for plugins not distributed via plugins catalog or in environments without internet connection.
 
-To uninstall a plugin:
+Download the archive containing the plugin assets and install it by extracting the archive into  plugin directory. For example:
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:122]: Download the archive containing the plugin assets, and install it by extracting the archive into the plugin directory. For example:

File: docs/sources/administration/plugin-management/index.md [hunk 6]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
...
+unzip my-plugin-0.2.0.zip -d YOUR_PLUGIN_DIR/my-plugin
+```
 
-1. In a web browser, navigate to the [Grafana plugin catalog](https://grafana.com/plugins) and find a plugin that you want to install.
-1. Click the plugin, and then click the **Installation** tab.
+The path to the plugin directory is defined in the configuration file. For more information, refer to [Configuration]({{< relref "../../setup-grafana/configure-grafana/#plugins" >}}).
 
-### Install plugin on Grafana Cloud
+#### Install a plugin in air-gapped environment
+Plugin installation usually requires an internet connection, you can check which endpoints are being used on your instance when installing a plugin regularly and add them to allowlist of your instance.
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:131]: Plugin installation usually requires an internet connection. You can check which endpoints are used during the installation on your instance and add them to your instance’s allowlist.

File: docs/sources/administration/plugin-management/index.md [hunk 7]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
...
+If this is not possible you can go via installing a plugin using [grafana cli](#install-a-plugin-using-grafana-cli) or as a [zip file](#install-a-plugin-from-a-zip-file).
 
-Grafana Cloud handles the plugin installation automatically.
+You can fetch any plugin from Grafana.com API following the download link referenced in the API.
+Here is an example based on `grafana-lokiexplore-app` plugins.
 
-If you're logged in to Grafana Cloud when you add a plugin, log out and then log back in again to use the new plugin.
+1. Open `https://grafana.com/api/plugins/grafana-lokiexplore-app` and look for `links` section 
+1. Find a `download` url which looks something like `https://grafana.com/api/plugins/grafana-lokiexplore-app/versions/1.0.2/download`
+1. Using this URL you can fetch the plugin zip file that you can as described above.
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:140]: 1. Use this URL to download the plugin ZIP file, which you can then install as described above.

File: docs/sources/administration/plugin-management/index.md [hunk 8]
```diff
@@ -162,21 +152,30 @@ plugins:
...
+To update a plugin:
 
-#### Install a packaged plugin
+1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
+1. Click the **Installed** filter to show only installed plugins.
+1. Click the plugin's logo.
+1. Click **Update**.
 
-After the user has downloaded the archive containing the plugin assets, they can install it by extracting the archive into their plugin directory. For example:
+When the update is complete, you'll see a confirmation message that the update was successful.
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:166]: When the update is complete, a confirmation message will indicate it was successful.

File: docs/sources/administration/plugin-management/index.md [hunk 9]
```diff
@@ -162,21 +152,30 @@ plugins:
   - redis-datasource
 ```
 
-### Install plugin on local Grafana
+When the update is complete, you'll see a confirmation message that the installation was successful.
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:155]: When the update is complete, a confirmation message will indicate it was successful.

**Review by irenerl24 — CHANGES_REQUESTED** [review]

File: docs/sources/administration/plugin-management/index.md [hunk 10]
```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
...
-1. Click the **Installed** filter to show only installed plugins.
-1. Click the plugin's logo.
-1. Click **Update**.
+#### Install a plugin using grafana cli
+Grafana CLI allows you to install, upgrade, and manage your Grafana plugins using a command line. For more information about Grafana CLI plugin commands, refer to [Plugin commands]({{< relref "../../cli/#plugins-commands" >}}).
 
-When the update is complete, you'll see a confirmation message that the update was successful.
 
-### Uninstall a plugin
+#### Install a plugin from a zip file
```
Comment (irenerl24) [inline docs/sources/administration/plugin-management/index.md:119]: #### Install a plugin from a ZIP file

**Review by irenerl24 — CHANGES_REQUESTED** [review]

File: docs/sources/administration/plugin-management/index.md (hunk 1 above)
Comment (tolzhabayev) [reply docs/sources/administration/plugin-management/index.md]: thanks, adjusted

File: docs/sources/administration/plugin-management/index.md (hunk 2 above)
Comment (tolzhabayev) [reply docs/sources/administration/plugin-management/index.md]: thanks, adjusted

File: docs/sources/administration/plugin-management/index.md (hunk 4 above)
Comment (tolzhabayev) [reply docs/sources/administration/plugin-management/index.md]: thanks, adjusted

File: docs/sources/administration/plugin-management/index.md (hunk 6 above)
Comment (tolzhabayev) [reply docs/sources/administration/plugin-management/index.md]: thanks, adjusted

File: docs/sources/administration/plugin-management/index.md (hunk 8 above)
Comment (tolzhabayev) [reply docs/sources/administration/plugin-management/index.md]: thanks, adjusted

File: docs/sources/administration/plugin-management/index.md (hunk 10 above)
Comment (tolzhabayev) [reply docs/sources/administration/plugin-management/index.md]: thanks, adjusted

File: docs/sources/administration/plugin-management/index.md (hunk 9 above)
Comment (tolzhabayev) [reply docs/sources/administration/plugin-management/index.md]: thanks, adjusted

**Review by irenerl24 — APPROVED** [review]

**Pull request merged**